aspnet/fundamentals/localization/sample/project.json

```diff
@@ -0,0 +1,54 @@
+﻿{
+  "userSecretsId": "aspnet5-Localization.StarterWeb-4772c50f-56f3-4058-bd61-75f1352aeca7",
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "EntityFramework.Commands": "7.0.0-*",
```

Comment:
@shirhatti  Also this is a pure copy of [entropy](https://github.com/aspnet/entropy)